caps/static/js/main.js

```diff
@@ -121,4 +121,25 @@ $(function() {
         );
     });
 
+    $("a").click(function(e) {
```

Comment:
A stylistic thing, but you probably want `.on('click', …)` here, rather than `.click()`.

I was originally going to say, binding to _all_ anchor elements on the site feels wasteful, but I bet it actually makes no difference in reality. If you wanted to only bind to links with a `plan-id` attribute, you could do it by selecting: `$('a[data-plan-id]')`. Then you wouldn’t need the `if` inside the callback.